docs/config.json

```diff
@@ -732,8 +732,42 @@
               "title": "Moderated Sessions",
               "slug": "/access-controls/guides/moderated-sessions/"
             }
-          ]
+          ] 
         },
+	{
+	    "title": "Set up Access Requests",
+	    "slug": "/access-controls/access-requests/",
+	    "entries": [
+		{
+		    "title": "Mattermost",
+		    "slug":"/access-controls/access-requests/ssh-approval-mattermost/"
+		},
+		{
+		    "title": "PagerDuty",
+		    "slug":"/access-controls/access-requests/ssh-approval-pagerduty/"
+		},
+                {
+                    "title": "Jira Server",
+                    "slug":"/access-controls/access-requests/ssh-approval-jira-server/"
+                },
+		{
+		    "title": "Jira Cloud",
+		    "slug":"/access-controls/access-requests/ssh-approval-jira-cloud/"
+		},
+		{
+		    "title": "Slack",
+		    "slug":"/access-controls/access-requests/ssh-approval-slack/"
+		},
+		{
+		    "title": "Role Requests",
```

Comment:
Should these last 2 go in a different section? The preceding items are all about the external plugins for handling access requests, and these two are more about the Teleport feature for access requests.

Reply:
Makes sense—I'll add an "Access Requests" section